go.mod

```diff
@@ -7,6 +7,11 @@ require (
 	github.com/spf13/viper v1.16.0
 )
 
+require (
+	github.com/aws/aws-sdk-go v1.45.3 // indirect
```

Comment:
explain what you mean by `//indirect`

Reply:
didnt code/comment this, but in go,  //indirect used to indicate that a particular dependency is indirect, meaning that it's not directly used by your code but is a dependency of another package that your code depends on.

Reply:
Got it. that's neat to know go auto indicates this!